assignments/number-functions/code.js

```diff
@@ -0,0 +1,10 @@
+const add = (a,b) => (a+b);
```

Comment:
Overall looks good. Note you don't need `()`s around the expression. So this could just be.

```suggestion
const add = (a, b) =>  a + b;
```

And spaces around the operators and after commas is good style. If you right click in the editor pane you'll get a contextual menu that has an item "Format document". You can use that to fix a lot of these formatting issues.